### PR TITLE
ENH display the logos

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds our partners to our index and cleans out some old cruft we do not use.

### CODE REVIEW

**Comment by beckermr** [comment]
Screen Shot 2020-12-02 at 9 50 20 AM

**Comment by beckermr** [comment]
@chrisburr @SylvainCorlay @conda-forge/core Does this seem ok to you all?

**Comment by SylvainCorlay** [comment]
Thanks for taking care of this! This looks great!

**Comment by isuruf** [comment]
Thanks for doing this. I wonder if we should have just Institutional partners (instead of our sponsors) in there and also divide them into `infrastructure` and `dev-support`

**Comment by beckermr** [comment]
> Thanks for doing this. I wonder if we should have just Institutional partners (instead of our sponsors) in there.

Do you mean for the title? Sure!

> also divide them into infrastructure and dev-support

Yep that is easy to do as well!

**Comment by kkraus14** [comment]
I believe we'd be interested to have NVIDIA included as well (if appropriate). Will submit a PR to add it to the yml.

**Comment by beckermr** [comment]
Nvidia would be great! Please make sure we are allowed to display the logo @kkraus14.

**Comment by beckermr** [comment]
Screen Shot 2020-12-02 at 11 24 28 AM

**Comment by beckermr** [comment]
@isuruf is this better?

**Comment by kkraus14** [comment]
> Nvidia would be great! Please make sure we are allowed to display the logo @kkraus14.

Checking internally on the process and will handle it. Will report back here once we have a green light.

**Comment by isuruf** [comment]
Looks good to me. Can you push the code?

**Review by CJ-Wright — APPROVED** [review]

**Comment by CJ-Wright** [comment]
Are we expecting Anaconda to also be listed under developer support as well?

**Comment by beckermr** [comment]
Ahhhh. Yes. I will fix that.

**Comment by beckermr** [comment]
OK I added a fiscal sponsor section too.

Screen Shot 2020-12-02 at 1 20 24 PM

**Comment by beckermr** [comment]
@kkraus14 Once you add the `fiscal support` to the nvidia PR and merge it, it will display.

**Comment by kkraus14** [comment]
> @kkraus14 Once you add the `fiscal support` to the nvidia PR and merge it, it will display.

Still working on getting approval for the NVIDIA logo as well.

**Comment by beckermr** [comment]
Any other comments @conda-forge/core?

**Comment by kkraus14** [comment]
> > @kkraus14 Once you add the `fiscal support` to the nvidia PR and merge it, it will display.
> 
> Still working on getting approval for the NVIDIA logo as well.

Had to change the URL used for the logo but just got approval so we should be good to go on the NVIDIA logo front.

**Comment by beckermr** [comment]
Great! Feel free to merge your PR at any time. I will resolve any conflicts and merge this one when it is ready.

**Comment by kkraus14** [comment]
> Great! Feel free to merge your PR at any time. I will resolve any conflicts and merge this one when it is ready.

Merged!

**Comment by jakirkham** [comment]
It looks like the `scripts` directory is getting deleted. Is that intended?

**Comment by beckermr** [comment]
> It looks like the scripts directory is getting deleted. Is that intended?

yes - we do not use it.

**Comment by isuruf** [comment]
Some of these scripts are supposed to be used locally. like `watch_only_my_feedstocks.py`. Not sure if this is used by anyone though

**Comment by beckermr** [comment]
I can restore if people feel strongly. It is confusing to others when old stuff is lying around that is not in use.

**Comment by beckermr** [comment]
I added back anything meant to be run locally. The rest of them appear to have been used to do things that are now done by other services.

**Comment by isuruf** [comment]
NumFOCUS is not an institutional partner, IMO. We are part of NumFOCUS.

**Comment by beckermr** [comment]
Fixed!

**Comment by beckermr** [comment]
OK. I am going to let this sit for a while. Then I will press merge!

File: scripts/whats_missing.py
Comment (jakirkham) [inline scripts/whats_missing.py:1]: For context, this was used to figure out what packages we might be missing in `conda-forge` that `defaults` has. Not sure if we have a replacement for this.

Comment (beckermr) [reply scripts/whats_missing.py:1]: ok will add back.

**Comment by jakirkham** [comment]
> I can restore if people feel strongly. It is confusing to others when old stuff is lying around that is not in use.

Not very strongly, but it does seem unrelated to this change.

**Comment by beckermr** [comment]
done.

**Review by kkraus14 — APPROVED** [review]

**Review by CJ-Wright — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

**Pull request merged**
